src/pyhf/optimize/opt_minuit.py

```diff
@@ -87,17 +87,22 @@ def _minimize(
         Minimizer Options:
             maxiter (:obj:`int`): maximum number of iterations. Default is 100000.
             return_uncertainties (:obj:`bool`): Return uncertainties on the fitted parameters. Default is off.
+            strategy: (:obj:`int`): See :attr:`iminuit.Minuit.strategy`. Default is to configure based on `do_grad`.
```

Comment:
```suggestion
            strategy: (:obj:`int`): See :attr:`iminuit.Minuit.strategy`. Default is to configure in response to `do_grad`.
```

Maybe slightly revise here? For whatever reason "based on" makes my mind think that this means "True -> True, False -> False" even though that isn't what that means. But just that this wasn't 100% clear to me right away makes me think it might be worth tweaking.